components/xip-patch/xip-patch.sh

```diff
@@ -86,6 +86,59 @@ EOF
     fi
 }
 
+requestGardenerCerts() {
+    local subdomain
+    local shoot_domain
+    local delay
+    subdomain="kyma"
+    delay=10
+
+    echo "Getting Shoot Domain"
+    shoot_domain="$(kubectl -n kube-system get configmap shoot-info -o jsonpath='{.data.domain}')"
+    DOMAIN="${subdomain}.${shoot_domain}"
+
+    echo "Requesting certificate for domain ${DOMAIN}"
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: cert.gardener.cloud/v1alpha1
+kind: Certificate
+metadata:
+  name: kyma-cert
+  namespace: kyma-installer
+spec:
+  commonName: "*.${DOMAIN}"
+EOF
+
+    while :
```

Comment:
Consider adding a condition to prevent the loop from iterating over and over again in case something goes wrong.

Reply:
Done